frontend/packages/jobs/src/trigger/jobs.ts

```diff
@@ -290,3 +291,7 @@ export const savePullRequest = async (
 export const helloWorld = async (name?: string) => {
   await helloWorldTask.trigger({ name: name ?? 'World' })
 }
+
+export const testSentryError = async () => {
+  await sentryErrorTest.trigger()
+}
```

Comment:
Please revert testSentryError, because I have confirmed to work 👍🏻 

https://cloud.trigger.dev/orgs/liam-hq-5035/projects/v3/liam-HdAt/runs/run_24up1dhexxc1fatgi2gnr?span=dddd7872da6aab75